Makefile

```
@@ -37,11 +38,17 @@ IMAGE_SUFFIX ?= $(strip )
 
 # REGISTRY ?= docker.io/library
 # Container registry for target images.
-REGISTRY ?= docker.io/library
+REGISTRY ?= docker.io/caicloud
 
-# Container registry for base images.
+# Container registry for build base images.
 BASE_REGISTRY ?= docker.io/library
 
+# Container registry for cyclone target base images.
+CYCLONE_BASE_REGISTRY ?= docker.io/caicloud
+
+# Version of the cyclone base images.
+CYCLONE_BASE_VERSION ?= v1.0.0
```

The prefix `CYCLONE_` is not necessary.

```suggestion
BASE_VERSION ?= v1.0.0
```

---

In order to keep consistent with `CYCLONE_BASE_REGISTRY`.